core/src/main/scala/org/apache/spark/deploy/security/HadoopDelegationTokenManager.scala

```diff
@@ -75,6 +78,17 @@ private[spark] class HadoopDelegationTokenManager(
       .toMap
   }
 
+  def getRenewableDelegationTokens(): Option[RenewableDelegationTokens] = {
```

Comment:
I'm not too fond of the idea of having two methods for creating tokens (this and `obtainDelegationTokens`). One should be enough.